src/main/java/com/descope/model/sso/RoleMapping.java

```diff
@@ -11,7 +11,6 @@
 @NoArgsConstructor
 @AllArgsConstructor
 public class RoleMapping {
-
   private List<String> groups;
-  private String role;
+  private String roleName;
```

Comment:
This will not be backward compatible right ?

Reply:
Yes, but it did not work before and no-one used it (other problems as well in the SAML).